internal/db/dao.go

```diff
@@ -82,6 +83,27 @@ func (dao *PostgresDAO) GetGoals(pagination Pagination, filter GetGoalsFilter) (
 	return list, nil
 }
 
+func (dao *PostgresDAO) GetGoal(id string) (top90.Goal, error) {
+
+	query := fmt.Sprintf("SELECT * FROM %s WHERE id is %s", tableNames.Goals, id)
```

Comment:
Did you test this? It doesn't seem to work... I don' think `is` is valid. It should probably be `=` and you probably need `` around the %s.